### PR TITLE
Update to php-cs-fixer ^2.0 release

### PR DESCRIPTION
**Pull request by raphaelstolt**
PHP CS Fixer `2.0.0` was released today.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,10 @@ env:
 
 matrix:
   include:
-    - php: hhvm
+    - php: hhvm-3.9
+      sudo: required
+      dist: trusty
+      group: edge
```
Comment (localheinz) [inline .travis.yml]: Ha, interesting - was wondering why builds on HHVM were failing after updating to `friendsofphp/php-cs-fixer:^2.0.0`, so this apparently helps?

File: .php_cs [hunk 2]
```diff
@@ -12,6 +12,6 @@ $cacheDir = getenv('TRAVIS') ? getenv('HOME') . '/.php-cs-fixer' : __DIR__;
 
 return PhpCsFixer\Config::create()
     ->setRules($rules)
-    ->finder($finder)
+    ->setFinder($finder)
     ->setUsingCache(true)
```
Comment (localheinz) [inline .php_cs]: Defaults to `true` now, see [`FriendsOfPHP\Config#L35`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.0.0/src/Config.php#L35).

File: composer.json [hunk 3]
```diff
@@ -24,8 +24,8 @@
     "ext-mbstring": "*"
   },
   "require-dev": {
-    "friendsofphp/php-cs-fixer": "2.0.0-alpha",
-    "phpunit/phpunit": "@stable"
+    "phpunit/phpunit": "@stable",
+    "friendsofphp/php-cs-fixer": "^2.0"
```
Comment (localheinz) [inline composer.json]: How about adding


```json
{
    "config": {
        "sort-packages": true
    }
}
```

to `composer.json` to keep packages sorted without the need to specify the `--sort-packages` option?

💁‍♂️ For reference, see https://getcomposer.org/doc/06-config.md#sort-packages.

File: .php_cs (hunk 2 above)
Comment (raphaelstolt) [inline .php_cs]: Good catch, will remove it later today.

File: composer.json (hunk 3 above)
Comment (raphaelstolt) [inline composer.json]: Totally :+1:,  but should be done in a separate PR. Will open one later today.

**Comment by rquadling** [comment]
We all happy with this one?

**Comment by rquadling** [comment]
Actually, all is fine!

**Pull request merged**